e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/add-payment.spec.js

```diff
@@ -135,7 +137,12 @@ context('PDC_RECONCILE users can add a payment to a report', () => {
     cy.contains('button', 'Continue').click();
 
     cy.contains('Premium payments').should('exist');
-    pages.utilisationReportPage.premiumPaymentsTab.getPremiumPaymentsTableRow(FEE_RECORD_ID_ONE).should('contain', 'MATCH');
+    pages.utilisationReportPage.premiumPaymentsTab.premiumPaymentsTable
+      .getStatus(FEE_RECORD_ID_ONE)
+      .invoke('text')
+      .then((text) => {
+        expect(text.trim()).to.equal('MATCH');
```

Comment:
Not a blocker for this PR (as this was already in place), seems this should be a constant (123 instances of this)

Reply:
I believe Christian and Abhi had a chat about this and my understanding is that a ticket is going to be raised to address this, so out of scope for this PR